website/client/package-lock.json

```diff
@@ -11676,7 +11676,7 @@
       }
     },
     "habitica-markdown": {
-      "version": "1.4.0",
```

Comment:
I think you'll have to manually install `habitica-markdown@2` with `npm i habitica-markdown@2` or the lockfile won't be updated properly